publications/joss/paper.bib

```diff
@@ -508,17 +508,6 @@ @ARTICLE{Iyer2023Sphinx
 
 
 
-@article{batalha2017pandexo,
```

Comment:
The metadata for references cited by JOSS articles is generated from the `.bib` file, rather than the `.md` file. As such, the bibliography file should only include references which are cited by the paper. I have removed two entries that didn't seem to be cited.